base_geoengine/doc/source/conf.py

```diff
@@ -11,23 +12,29 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
+import sys
+import os
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/'))
-sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/'))
-sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/addons/'))
+# sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/'))
+sys.path.insert(
+    0,
+    os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/'))
+sys.path.insert(
+    0,
+    os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/addons/'))
```

Comment:
This should be fixed on the 6.1 and then forward ported into the others branches